Formula/docker-machine-driver-hyperkit.rb

```diff
@@ -0,0 +1,35 @@
+class DockerMachineDriverHyperkit < Formula
+  desc "Docker Machine driver for hyperkit"
+  homepage "https://github.com/kubernetes/minikube"
+  url "https://github.com/kubernetes/minikube.git",
+    :tag => "v0.27.0",
+    :revision => "a7c0cbee11dd97f257cc8d050605684c0c446a7e"
+
+  depends_on "go" => :build
+  depends_on :macos => :yosemite
+  depends_on "docker-machine" => :recommended
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "1"
```

Comment:
This is the default; doesn't need writing.

Reply:
Thanks, updated. 👍